python/pypio/pypio.py

```diff
@@ -15,9 +15,35 @@
 # limitations under the License.
 #
 
+from __future__ import absolute_import
+
+import sys
+
 from pypio.data import PEventStore
-from pypio.workflow import CleanupFunctions
+from pyspark.sql import SparkSession
+from pyspark.sql import utils
+
+
+def init():
+    global spark
+    spark = SparkSession.builder.getOrCreate()
+    global sc
+    sc = spark.sparkContext
+    global sqlContext
+    sqlContext = spark._wrapped
+    global p_event_store
+    p_event_store = PEventStore(spark._jsparkSession, sqlContext)
+    print("Initialized pypio")
+
+
+def find(app_name):
+    return p_event_store.find(app_name)
 
-p_event_store = PEventStore(spark._jsparkSession, sqlContext)
-cleanup_functions = CleanupFunctions(sqlContext)
 
+def save(model):
```

Comment:
`saveModel` might be better as same as above.

Reply:
Fixed it.